cypress/pages/saucedemo/login.js

```diff
@@ -17,6 +18,9 @@ class LoginPage {
         this.elements.loginButton().click()
     }
 
+    createErrorMessage() {
```

Comment:
cuidado con el nombre del método!
Si lo leemos, suena a que vamos a crear un mensaje de error con el login page
yo pondría algo mas como:
```suggestion
    getErrorMessage() {
```